packages/desktop-client/src/components/reports/chart-theme.ts

```diff
@@ -14,15 +21,42 @@ const letterSpacing = 'normal';
 const fontSize = 13;
 
 // Labels
-const baseLabelStyles = {
+type LabelStyles = {
+  fontFamily: string;
+  fontSize: number;
+  letterSpacing: string;
+  fill: string;
+  stroke: string;
+};
+
+const baseLabelStyles: LabelStyles = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏same issue here
```suggestion
const baseLabelStyles = {
```

Reply:
Removed it